### PR TITLE
Curtis seeders

### PR DESCRIPTION
**Pull request by Coline1186**

### CODE REVIEW

**Review by maximmft — APPROVED** [review]

File: client/src/components/Card/Card.jsx [hunk 1]
```diff
@@ -1,54 +1,86 @@
...
+          <FaRegHeart className="icon" />
+        </div>
+        <div>
+          <SlOptionsVertical onClick={handleCard} className="icon" />
+        </div>
+      </div>
+      <div className="card-title">
+        <p className="title">{product.name}</p>
+        <div className="price-and-logo">
+          <HiOutlineShoppingBag className="panier" />
```
Comment (VinchC) [inline client/src/components/Card/Card.jsx]: English

File: client/src/components/Nav/SearchInput.jsx [hunk 2]
```diff
@@ -1,17 +1,66 @@
...
+import SearchResult from "./SearchResult";
 
 function SearchInput({ setShowInput }) {
+
+  const port =import.meta.env.VITE_API_URL
+  const [inputValue, setInputValue] = useState("");
+  const [results, setResults] = useState([]);
+  const handleInput = (e) => {
+    setInputValue(e.target.value);
+    // handleClick()
```
Comment (VinchC) [inline client/src/components/Nav/SearchInput.jsx]: delete comment

File: client/src/components/Nav/SearchResult.jsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    </div>
+  );
+}
+
+SearchResult.propTypes = {
+  setShowInput: PropTypes.func.isRequired,
+  results: PropTypes.arrayOf(
+    PropTypes.shape({
+      Id_product: PropTypes.number.isRequired,
+      // Add other product properties here as needed
```
Comment (VinchC) [inline client/src/components/Nav/SearchResult.jsx]: delete comment?

File: server/app/controllers/wishListActions.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+     const showIt = await table.wish_list.getIt(parseId);
+     res.status(200).json(showIt)  
+    } catch (error) {
+        next(error)
+    }
+}
+
+const remove = async (req, res, next) => { 
+    try {
+        // eslint-disable-next-line camelcase
```
Comment (VinchC) [inline server/app/controllers/wishListActions.js]: delete disable

**Review by VinchC — CHANGES_REQUESTED** [review]

File: server/app/controllers/wishListActions.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
+const table = require("../../database/tables")
+
+const add = async(req,res,next)=>{
+
+    // eslint-disable-next-line camelcase
```
Comment (VinchC) [inline server/app/controllers/wishListActions.js]: delete disable

**Review by VinchC — APPROVED** [review]

**Pull request merged**